core/src/main/java/com/novoda/landingstrip/LandingStrip.java

```diff
@@ -134,79 +127,18 @@ public void scrollTo(int x, int y) {
     }
 
     @Override
-    protected void onAttachedToWindow() {
-        super.onAttachedToWindow();
-        pagerAdapterObserver.reregister();
+    public void onPageScrolled(int position, float positionOffset, int positionOffsetPixels) {
+        scrollingPageChangeListener.onPageScrolled(position, positionOffset, positionOffsetPixels);
```

Comment:
not sure I follow, it is implementing the interface but the implementation is extracted?

Or do you mean exposing the  `ScrollingPageChangeListener` as a public api instead of setting the 
`LandingStrip` as a page change listener?